src/devdocs2zim/generator.py

```diff
@@ -363,6 +373,27 @@ def load_common_files(self) -> list[StaticItem]:
             )
         )
 
+        # Dynamic navbar
```

Comment:
That make sense! I also added a check that index.js exists in it because that file is directly linked. I think it would be too easy otherwise to build a non-working Zim.